pennylane/math/multi_dispatch.py

```diff
@@ -870,6 +870,49 @@ def norm(tensor, like=None, **kwargs):
     return norm(tensor, **kwargs)
 
 
+@multi_dispatch(argnum=[0])
+def svd(tensor, like=None, **kwargs):
+    """Compute the singular value decomposition of a tensor in each interface.
+    For a matrix A, the singular value decomposition consist of three matrices U, S and Vh, such that:
+
+    .. math::
+
+        A = U . Diag(S) . Vh
+
+    Returns:
+        S, U, V: full decomposition (if compute_uv is True or None)
+        S: only the singular values (if compute_uv is False)
+
+    """
+    if like == "tensorflow":
+        from tensorflow.linalg import svd, adjoint
+
+        # Tensorflow results need some pos-processing to keep it similar to other frameworks.
+        if kwargs.get("compute_uv", True):
+            S, U, V = svd(tensor, **kwargs)
+            return U, S, adjoint(V)
+        return svd(tensor, **kwargs)
+
+    if like == "jax":
+        from jax.numpy.linalg import svd
+
+    elif like == "torch":
+        # Torch is deprecating torch.svd() in favor of torch.linalg.svd().
+        # The new UI is slightly different and breaks the logic for the multi dispatching.
+        # This small workaround restores the compute_uv control argument.

```

Comment:
If handling `compute_uv` is problematic, it will be fine to not have it at all.

Reply:
It is not problematic, only needs some work. We can keep the full functionality with this workaround.